components/operator/internal/state/registry.go

```diff
@@ -67,6 +70,7 @@ func addRegistryConfigurationWarnings(s *systemState) {
 }
 
 func setInternalRegistryConfig(ctx context.Context, r *reconciler, s *systemState) error {
+	r.Event(&s.instance, internalRegistryEventType, internalRegistryEventReason, internalRegistryEventMessage)
```

Comment:
This event will be emitted during every reconciliation if the internal docker registry is enabled. Is it expected behavior?

```
Warning  InternalDockerRegistryEnabled  5m (x9 over 5m36s)  serverless-operator  Internal Docker Registry is not Highly Available registry and should be used for development purpose only
```

Reply:
I'm not sure if there is other way. Lets chat